packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -188,8 +188,8 @@ const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
     })(),
     padding:
       props.appearance === 'quiet'
-        ? designTokens.paddingForInputAsQuiet
-        : designTokens.paddingForInput,
+        ? '0 var(--spacing-20)'
+        : '0 var(--spacing-30)',
```

Comment:
Please use de designTokens from the `designTokens` object.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/098974e6596998284f5d665ce8a0a416310509e0
Thank you

Reply:
These one haven't been updated.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/065e27db2ab90e9c6103bb78679b8c7d9ee5283f
Thank you